submissions/Yasya23/tiny-js-world/index.js

```diff
@@ -0,0 +1,75 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  species: "dog",
+  name: "Bob",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof",
+};
+
+const cat = {
+  species: "cat",
+  name: "Grey",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "meow",
+};
+
+const man = {
+  species: "human",
+  name: "Anton",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "Have a good day!",
+};
+
+const woman = {
+  species: "human",
+  name: "Anna",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "Good luck!",
+};
+
+const catWoman = { ...cat };
+catWoman.name = "Luci";
+catWoman.gender = "female";
+
+const inhabitants = [dog, cat, man, woman, catWoman];
+const properties = ["species", "name", "gender", "legs", "hands", "saying"];
+
+inhabitants.forEach(function (elem) {
```

Comment:
Here we can safely use an arrow function. Less letters - easier to read.

`elem` is a generic term.
From the checklist:
> Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose.

Reply:
Thanks for checking! Fixed it.